### ISSUE TITLE
Jumptables detected as functions in twain_32.dll

### ISSUE THREAD

**Issue by pinkflawd** [radare/radare2#7435]
When analyzing twain_32.dll with the config anal.hasnext = true, a lot more legitimate functions are detected than without; however also a jumptable is analyzed as a function. See below, 0x66204a6d is the jumptable offset.

Have seen this happen with a number of binaries & jumptables more.

```
[0x6620b550]> pd 10 @ 0x66204a6d
/ (fcn) fcn.66204a6d 80
|   fcn.66204a6d ();
|              ; JMP XREF from 0x66204971 (fcn.6620494d)
|              ; DATA XREF from 0x66204971 (fcn.6620494d)
|       ,=< 0x66204a6d      7849           js 0x66204ab8
|       |   0x66204a6f      206697         and byte [esi - 0x69], ah
|       |   0x66204a72      49             dec ecx
|       |   0x66204a73      20669e         and byte [esi - 0x62], ah
|       |   0x66204a76      49             dec ecx
|       |   0x66204a77      2066a5         and byte [esi - 0x5b], ah
|       |   0x66204a7a      49             dec ecx
|       |   0x66204a7b      2066ac         and byte [esi - 0x54], ah
|       |   0x66204a7e      49             dec ecx
|       |   0x66204a7f      2066b3         and byte [esi - 0x4d], ah
[0x6620b550]> axt @ 0x66204a6d
code 0x66204971 jmp dword [eax*4 + fcn.66204a6d] in fcn.6620494d
data 0x66204971 jmp dword [eax*4 + fcn.66204a6d] in fcn.6620494d
```

password infected
[twain_32.dll.zip](https://github.com/radare/radare2/files/981158/twain_32.dll.zip)

**Comment by pinkflawd** [comment]
Same case at 0x66203e5d

**Comment by alvarofe** [comment]
Try with `e anal.jmptbl=true` is gonna spit r2 commands. This was an experiment by pancake that must be improved. I will try to take a look in the following days since I need this as well for other stuff. However, 0x66204a6d is still a function.

![image](https://cloud.githubusercontent.com/assets/3474042/25776165/6d860c78-32b7-11e7-9e7a-d7d05acf102e.png)

**Comment by pinkflawd** [comment]
Yup, tried jmptbl, doesn't change the issue, ```e anal.hasnext = true``` is the troublemaker in this case :)

**Comment by trufae** [comment]
The analysis issue for this binary has been fixed already, i have tried with hasnext enabled and disabled, and in both cases the control flow is correct. We will need to add 2 tests for this (with and without hasnext), but i have some questions here:

* Why a 56KB binary takes so much time to load? 
  * @pinkflawd would you like to submit a PR adding those tests? I can help you on any question if needed
* I understand this binary comes from Windows and we can't distribute it in the testsuite, so which is the right solution?

Thanks for reporting!!

Also i have the idea that if we submit and associate tests to specific issues it will be possible to automatically add feedback to the issues when the specific test is fixed. because like in this case, the analysis bug was fixed by fixing another problem
